app/src/main/java/org/sopt/dosopttemplate/LoginActivity.kt

```diff
@@ -0,0 +1,50 @@
+package org.sopt.dosopttemplate
+
+import android.content.Intent
+import android.os.Bundle
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import com.google.android.material.snackbar.Snackbar
+import org.sopt.dosopttemplate.databinding.ActivityLoginBinding
+
+class LoginActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityLoginBinding
+    override fun onCreate(savedInstanceState: Bundle?) {
+        binding = ActivityLoginBinding.inflate(layoutInflater)
+        super.onCreate(savedInstanceState)
+        setContentView(binding.root)
+
+        // 회원가입 하러 가기
+        binding.btnSignupSignup.setOnClickListener {
+            val intent = Intent(this, SignUpActivity::class.java)
+            startActivity(intent)
+            // finish()
+        }
+
+        val getId = intent.getStringExtra("ID")
+        val getPw = intent.getStringExtra("PW")
+        val getNickname = intent.getStringExtra("Nickname")
+        val getAge = intent.getStringExtra("Age")
+
+        // 로그인 하기
+        binding.btnLoginLogin.setOnClickListener {
+            if (binding.etSignupId.text.toString() == getId && binding.etSignupPw.text.toString() == getPw) {
+                val toast = Toast.makeText(applicationContext, getString(R.string.login_success), Toast.LENGTH_SHORT)
+                toast.show()
```

Comment:
toast는 해당 상황에서만 사용되는 일회성이기 때문에, 굳이 val로 만들지 않고 한번에
```suggestion
Toast.makeText(applicationContext, getString(R.string.login_success), Toast.LENGTH_SHORT).show() 
```
형태로 작성하시는게 더 좋을 듯 합니다 ~~